content/doc/book/installing/_setup-wizard.adoc

```diff
@@ -36,6 +36,7 @@ image:tutorials/setup-jenkins-01-unlock-jenkins-page.jpg[alt="Unlock Jenkins pag
 image:tutorials/setup-jenkins-02-copying-initial-admin-password.png[alt="Copying initial admin password",width=80%] +
   *Note:* 
 * The command: `sudo cat /var/lib/jenkins/secrets/initialAdminPassword` will print the password at console. +
+* If you running Jenkins by a docker container you can use `sudo docker exec jenkins-blueocean cat /var/jenkins_home/secrets/initialAdminPassword` to print the password at console without get inside it. The `jenkins-blueocean` it is the link:http://localhost:4242/doc/book/installing/#downloading-and-running-jenkins-in-docker[container-name user in our docs sample], just remember to replace it to your own if you pick a different one. +
```

Comment:
This `to print the password at console without get inside it.` should be `to print the password in the console without having to exec into the container.`